Action/Rest/GetAllAction.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Pim\Bundle\CustomEntityBundle\Action\Rest;
+
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
```

Comment:
I'd like to have a short description of the class responsability

Reply:
Sure, I'll add in that description